### PR TITLE
Github template - Add some notes for new profile and plugins

### PR DESCRIPTION
**Pull request by Gustry**
Some feedbacks from issues #44438 #44432 today

Preview on https://github.com/Gustry/QGIS/issues/new/choose

@gioman What do you think ?

### CODE REVIEW

**Comment by gioman** [comment]
@Gustry yes please!

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 1]
```diff
@@ -38,6 +43,18 @@ body:
     validations:
       required: true
 
+  - type: checkboxes
+    id: new-profile
+    attributes:
+      label: New profile
+      description: |
+        Did you try with a new QGIS Profile ? Some issues or crash might be related to plugins or user configuration.
```
Comment (agiudiceandrea) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:51]:
```suggestion
        Did you try with a new QGIS Profile? Some issues or crashes might be related to plugins or user configuration.
```

Comment (nirvn) [reply .github/ISSUE_TEMPLATE/10_bug_report.yml:51]: Can we decapitalize "profile", there is no reason for it to have a capital P.

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 2]
```diff
@@ -8,6 +8,11 @@ body:
       value: |
         Thanks for taking the time to fill out this bug report correctly.
 
+        Please report only bug related to QGIS Desktop or Server.
+
+        If it's related to a plugin, you should fill the issue in the plugin code repository.
```
Comment (nyalldawson) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:13]:
```suggestion
        If it's related to a plugin, you should file the issue in the plugin code repository.
```

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 3]
```diff
@@ -8,6 +8,11 @@ body:
       value: |
         Thanks for taking the time to fill out this bug report correctly.
 
+        Please report only bug related to QGIS Desktop or Server.
+
+        If it's related to a plugin, you should file the issue in the plugin code repository.
```
Comment (m-kuhn) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:13]:
```suggestion
        If your issue is related to a plugin, you should file the issue in the plugin code repository.
```

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 4]
```diff
@@ -8,6 +8,11 @@ body:
       value: |
         Thanks for taking the time to fill out this bug report correctly.
 
+        Please report only bug related to QGIS Desktop or Server.
```
Comment (m-kuhn) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:11]:
```suggestion
        Please report only issues related to QGIS.
```

... there's also qgis_process, not sure we want to go into that much detail here

**Comment by Gustry** [comment]
Gentle ping

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**